build.gradle

```diff
@@ -58,7 +58,10 @@ subprojects {
         options.compilerArgs << "-Xlint:deprecation"
         options.errorprone.disableAllWarnings = true
         options.errorprone.disableWarningsInGeneratedCode = true
-        options.errorprone.error("StringCaseLocaleUsage")
+        options.errorprone.error(
```

Comment:
This change needs to go into `build-logic/src/main/kotlin/polaris-java.gradle.kts` now (right at the top)

Reply:
Thanks for your review. Updated `polaris-java.gradle.kts` instead.